### ISSUE TITLE
Accessing an element that persists across pages triggers staleness deprecation warning

### ISSUE THREAD

**Issue by Castone22** [watir/watir#830]
## Meta -
Watir Version: 6.15.1
Selenium Version: 3.141.0
Browser Version: 71.0.3578.80 (Official Build) (64-bit)
Browser Driver Version: ChromeDriver 2.44.609538 (b655c5a60b0b544917107a59d4153d4bf78e1b90)
OS Version: Windows 10 Enterprise 1709-16299.785

## Expected Behavior -
As a tester, when i interact with an element that persists across pages, I expect it to react to the command i gave it.

Example:
  Given I visit google 
  And have searched for "Hello World"
  When I enter "Goodbye World" into the search box
  Then I should be displayed the appropriate results.

## Actual Behavior -
"When I enter "Goodbye world" into the search box" fails catastrophically with the stale_exists deprication

The key here is that the @search_box_element is persisting across steps and doms.

## Steps to reproduce -
Open IRB

```ruby
require 'Watir'
@browser = Watir::Browser.new :chrome
@browser.goto 'google.com'
search_box = @browser.text_field(title: 'Search')
search_box.set('Hello World'); search_box.send_keys(:enter)
search_box.set('Goodbye World')
> 2018-12-12 14:16:28 WARN Watir [DEPRECATION] ["stale_exists"] Checking `#exists? == false` to determine a stale element is deprecated. Use `#stale? == true` instead; see explanation for this deprecation: http://watir.com/staleness-changes.
> 2018-12-12 14:16:28 WARN Watir [DEPRECATION] ["stale_exists"] Checking `#exists? == false` to determine a stale element is deprecated. Use `#stale? == true` instead; see explanation for this deprecation: http://watir.com/staleness-changes.
> 2018-12-12 14:16:28 WARN Watir [DEPRECATION] ["stale_exists"] Checking `#exists? == false` to determine a stale element is deprecated. Use `#stale? == true` instead; see explanation for this deprecation: http://watir.com/staleness-changes.
> 2018-12-12 14:16:28 WARN Watir [DEPRECATION] ["stale_exists"] Checking `#exists? == false` to determine a stale element is deprecated. Use `#stale? == true` instead; see explanation for this deprecation: http://watir.com/staleness-changes.
> 2018-12-12 14:16:28 WARN Watir [DEPRECATION] ["stale_exists"] Checking `#exists? == false` to determine a stale element is deprecated. Use `#stale? == true` instead; see explanation for this deprecation: http://watir.com/staleness-changes.
```

Throws this until i kill the irb prompt.

**Comment by Castone22** [comment]
It's worth noting here, I'm *not* testing for a stale element in this particular case, i'm assuming that watir would continue to handle the fact that the element became stale when the page changed internally just like it did pre refactor.

**Comment by titusfortner** [comment]
yeah, looks like an over-aggressive deprecation warning. Your code will work as expected after the code is updated. I'll see if I can minimize the scope of the warning.

**Comment by titusfortner** [comment]
Also it shouldn't "fail catastrophically," just throw a deprecation warning.

**Comment by Castone22** [comment]
there's a little more than that, it throws the deprecation in a loop, and never actually sets the field.

**Comment by Castone22** [comment]
```
require 'Watir'; @browser = Watir::Browser.new :chrome; @browser.goto 'google.com'; search_box = @browser.text_field(title: 'Search'); search_box.set('Hello World'); search_box.send_keys(:enter); search_box.set('Goodbye World')
```

updated the issue to clarify.

**Comment by PanoramicPanda** [comment]
Deprecation warning is spit out in the loop until Watir's timeout, after which:

`Watir::Exception::UnknownObjectException: timed out after 30 seconds, waiting for #<Watir::TextField: located: true; {:title=>"Search", :tag_name=>"input"}> to be located`

**Comment by Castone22** [comment]
one more thing, if i manually relocate the element it behaves like it did in 6.8.
```
search_box.locate.set('goodbye world')
```

**Comment by titusfortner** [comment]
Oh, yes, I see what you are saying, thanks for clarifying. Yes, this bug was introduced in 6.11, and is the same as #814. It is fixed in master, and will work with 6.16
